Aplib.Tests/Belief/BeliefTests.cs

```diff
@@ -48,8 +48,8 @@ public void Belief_AssignedToObservationType_IsCorrectlyImplicitlyConvertedToObs
     public void UpdateBelief_ShouldUpdateConditionIsSatisfied_UpdatesObservation()
     {
         // Arrange
-        List<int> list = [];
-        Belief<List<int>, int> belief = new(list, reference => reference.Count, AlwaysUpdate);
+        System.Collections.Generic.List<int> list = [];
+        Belief<System.Collections.Generic.List<int>, int> belief = new(list, reference => reference.Count, AlwaysUpdate);
```

Comment:
Dit kan sowieso wel in een `using System.Collections.Generic`.

Reply:
Daar heb je gelijk in, ik pas het aan